lib/Album/AlbumMapper.php

```diff
@@ -110,6 +110,23 @@ public function getForUser(string $userId): array {
 			return new AlbumInfo((int)$row['album_id'], $userId, $row['name'], $row['location'], (int)$row['created'], (int)$row['last_added_photo']);
 		}, $rows);
 	}
+	
+	/**
+	 * @param string $albumName
+	 * @return AlbumInfo
+	 */
+	public function getByName(string $albumName): ?AlbumInfo {
```

Comment:
This also need the user name as two user can have the same album name.

Reply:
Right you are. Apologies, that functionality got dropped when `getByName` was added. Now corrected.